README.rst

```diff
@@ -34,6 +34,7 @@ found inside them and so on::
    -h, --help             show this help message and exit
    -o, --overview         show (possibly long) overview of TNEF file contents
    -a, --attachments      extract attachments, by default to current dir
+   -z, --zip              extract attachments into a single zip file, by default to current dir
    -p PATH, --path PATH   optional explicit path to extract attachments to
```

Comment:
As the `--zip` option can be combined with the `--path` option, do we want to keep the wording here? ie the "extract"? Or should we change it  to "optional explicit path where to save attachments" or similar?

Reply:
"with --attachment or --zip, extract to given path instead of current dir" ?